tests/testthat/test-betareg.R

```diff
@@ -37,12 +41,20 @@ if (Sys.getenv("USER") != "travis") {
     test_that("p_value", {
       expect_equal(
         p_value(m1)$p,
-        c(0, 0, 0, 0, 0, 0, 0, 0, 1e-05, 0.00114, 0, 6e-05),
+        c(
+          3.44252055464711e-250, 2.59222187511485e-65, 3.33610354513962e-29,
```

Comment:
why did you add this precision?

Reply:
Because the tests were failing with `testthat 3.0.0` and doing this fixed it (at least on my Windows machine).

Reply:
Ok, got it. But maybe you should not skip them everywhere? It should at least run on GitHub-CI.